spec/core_functions/string/split.hrx

```diff
@@ -0,0 +1,179 @@
+<===> options.yml
+---
+:ignore_for:
+- libsass
+
+<===>
+================================================================================
+<===> empty/input.scss
+a {b: split("")}
+
+<===> empty/output.css
+a {
+  b: [""];
+}
+
+<===>
+================================================================================
+<===> separator/input.scss
+a {b: split("a, b, c", ", ")}
+
+<===> separator/output.css
+a {
+  b: ["a", "b", "c"];
+}
+
+<===>
+================================================================================
+<===> empty_separator/input.scss
+a {b: split("Helvetica", "")}
+
+<===> empty_separator/output.css
+a {
+  b: ["H", "e", "l", "v", "e", "t", "i", "c", "a"];
+}
+
+<===>
+================================================================================
+<===> separator_not_found/input.scss
+a {b: split("a, b, c", "&")}
+
+<===> separator_not_found/output.css
+a {
+  b: ["a, b, c"];
+}
+
+<===>
+================================================================================
+<===> limit/input.scss
+a {b: split("a, b, c, d", ", ", 2)}
+
+<===> limit/output.css
+a {
+  b: ["a", "b", "c, d"];
+}
+
+<===>
+================================================================================
+<===> limit_zero/input.scss
+a {b: split("a, b, c", ", ", 0)}
```

Comment:
This seems like it should just be an error.

Reply:
Agreed. A `0` limit now triggers an error.